meeting-minutes/client-minutes/feb3-client-minutes.md

```diff
@@ -0,0 +1,58 @@
+
+### NOTES
+
+Since Pace B was waiting for a group member to join the call, and Pace A's client liaison hadn't joined the call yet, the client asked what Pace C had done during the week. Our github project board was shown so that the client could see which tickets we completed from the previous sprint, as well as what we plan to accomplish for the current sprint. The app was then run so that the client could visually see our progress, and she was impressed with what the team had done since last week. The client mentioned that we should keep in mind that Mentees should also be associated with an organization so that they can be matched with a mentor from that organization, and so I showed the client that that is something we're planning to incorporate after this week's sprint. She seemed impressed that we had thought of that as well. There were no suggestions for things to change.
+
+To address the roadblocks from above:
+
+1. After letting the client know of our current roadblock with the availability, unfortunately, the ability for mentees to book timeslots with mentors seems to be important to the client. She mentioned that ideally, Mentee's would receive a link to the Mentor's calendar in an email.
+
+- Given this, I asked if instead of our previous user flow (where Mentors create their calendar after registering, with our application managing user's calendars) if it would be possible to require Mentors to create their own calendars external of our application. The calendar process would then be handled outside of our application, BUT, as a part of the Mentor form, we could require Mentors via an input field to provide a link to whichever calendar platform they prefer. This link could then be sent to the Mentee in an email. That Mentee would then have access to the Mentor's calendar (For clarity: If I'm registering as a Mentor, I could create a calendar of my availability using an external calendar platform. I would copy the link to this external platform's calendar with my availability, and include that in an input field when I create my account on the Mentor Form. This link would then be sent to whatever Mentee that I accept the request for).
+- The client stated that this work-around would be acceptable. She notes that if we have time, we could continue to see if there's any other way. I mentioned that given the time remaining for the project, we might not be able to circle back to this issue to see if another platform would be free for us to use. So, overall, the way that I proposed today should be alright if we're unable to figure something else out before project completion.
+  - Please note: This work-around only occurred to me while talking with the client today. I wish I would have had time to propose this to the team beforehand, but I was thinking out loud.
```

Comment:
I thought I'd include a comment about this so it's easier to discuss:
Yes, I am aware that there could be a security risk behind this. For example, if we don't provide a protective measure, what's to stop a user from saying that the link to their calendar is [this](https://www.youtube.com/watch?v=xvFZjo5PgG0)? (I promise I'm not linking to to anything harmful, just a somewhat predictable youtube link). As a mentee, that'd be pretty useless.
I propose we: We move ahead with adding an input field to the Mentor form so that users can provide a link to an external calendar. BUT, we restrict acceptable links to be only Google Calendars for the time being. This somewhat disrupts the user flow (since Mentors would have to create a Google Calendar with their availability before successful account creation), but this is a workable solution for now. Given the time remaining for the project, and that the ability for Mentees to view Mentor's calendars being important feature to the client, I believe this is the best way to go.
@jpabadir @aleksssk @wrc9817 Please let me know your thoughts. If this is alright, I could get this setup for next week.

Reply:
I think restricting the links to only a Google calendar link is good idea!